src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -16,11 +16,12 @@
 
 namespace Xamarin.Android.Tasks
 {
-	public enum AotMode
+	public enum AotMode : uint
```

Comment:
Why?

Reply:
Because I want it to be unsigned... I am really uncomfortable with the (ab)use of `int` in .NET code - it invites all kinds of problems in situations where the value set should not (or does not, for whatever reason) allow negative integers.